build/bazel/remote/asset/v1/qualifiers.md

```diff
@@ -5,7 +5,27 @@ This lexicon defines standard qualifier names that servers
 
 The following standard qualifier `name`s are defined:
 
-* `resource_type`: This describes the type of resource.
+* `resource_type`: This describes the type of resource.  In case of file assets use
+  an existing [media type](https://www.iana.org/assignments/media-types/media-types.xhtml).
```

Comment:
Maybe reword as "File assets may use an existing media type ..." ?

Reply:
Thanks, tweaked the wording